packages/components/src/2016-12-01/button.js

```diff
@@ -27,13 +28,75 @@ class Button extends HTMLElement {
     shadowRoot.innerHTML = `
       <orion-inline
         border-radius="2"
-        background="black"
-        color="white"
+        background="white"
+        color="black"
         padding-horizontal="3"
         padding-vertical="2"
-        dim
         pointer><slot /></orion-inline>
     `;
+    this.shadowEl = shadowRoot.children[0];
+
+    this.addEventListener('mouseenter', () => {
+      if (this.state.disabled) { return; }
+      const nextState = ButtonState.enterHover(this.state);
+      this.dispatchEvent(new CustomEvent('change', { detail: nextState }));
```

Comment:
This is the flow I heard based on our conversation (start with the user clicking next month):
![flow](https://cloud.githubusercontent.com/assets/6705/21737763/22a5435c-d431-11e6-8224-69be0ff09018.png)